client/extensions/wp-super-cache/easy-tab.jsx

```diff
@@ -114,11 +113,15 @@ class EasyTab extends Component {
 					</form>
 				</Card>
 
-				{ phpModRewriteMessage &&
+				{ php_mod_rewrite &&
 				<Notice
 					className="wp-super-cache__notice-hug-card"
 					showDismiss={ false }
-					text={ phpModRewriteMessage } />
+					text={ translate(
+						'PHP caching enabled but Supercache mod_rewrite rules detected. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'PHP caching is enabled but Supercache mod_rewrite rules were detected. Cached files will be served using those rules. If your site is working ok, please ignore this message. Otherwise, you can edit the .htaccess file in the root of your install and remove the SuperCache rules.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).